__mocks__/fetch.js

```diff
@@ -0,0 +1,3 @@
+// Purposefully no-op here so tests will never actually make network requests. It's expected that tests will override
```

Comment:
I _think_ there's a Jest thing you can set in a jest.config.js file to automatically load this instead of having to do `jest.mock('./fetch.js') in the test file but I didn't bother with it, mostly out of laziness. But if there will be multiple test files, probably worth doing.